tests/foreman/cli/test_puppetclass.py

```diff
@@ -22,33 +22,31 @@
 from robottelo.cli.puppet import Puppet
 from robottelo.config import settings
 from robottelo.constants.repos import CUSTOM_PUPPET_REPO
-from robottelo.test import CLITestCase
-
-
-class PuppetClassTestCase(CLITestCase):
-    """Implements puppet class tests in CLI."""
-
-    @classmethod
-    @pytest.mark.skipif(not settings.repos_hosting_url)
-    def setUpClass(cls):
-        """Import a parametrized puppet class."""
-        super().setUpClass()
-        cls.puppet_modules = [{'author': 'robottelo', 'name': 'generic_1'}]
-        cls.org = make_org()
-        cv = publish_puppet_module(cls.puppet_modules, CUSTOM_PUPPET_REPO, cls.org['id'])
-        cls.env = Environment.list({'search': 'content_view="{}"'.format(cv['name'])})[0]
-        cls.puppet = Puppet.info(
-            {'name': cls.puppet_modules[0]['name'], 'environment': cls.env['name']}
-        )
-
-    @pytest.mark.tier2
-    @pytest.mark.upgrade
-    def test_positive_list_smart_class_parameters(self):
-        """List smart class parameters associated with the puppet class.
-
-        :id: 56b370c2-8fc6-49be-9676-242178cc709a
-
-        :expectedresults: Smart class parameters listed for the class.
-        """
-        class_sc_parameters = Puppet.sc_params({'puppet-class': self.puppet['name']})
-        self.assertGreater(len(class_sc_parameters), 0)
+
+
+@pytest.fixture(scope='module')
+def make_puppet():
+    """Import a parametrized puppet class."""
+    puppet_modules = [{'author': 'robottelo', 'name': 'generic_1'}]
+    org = make_org()
+    cv = publish_puppet_module(puppet_modules, CUSTOM_PUPPET_REPO, org['id'])
+    env = Environment.list({'search': 'content_view="{}"'.format(cv['name'])})[0]
```

Comment:
@vsedmik We will be changing our code standard soon, see https://github.com/SatelliteQE/robottelo/pull/8127/files 
It's recommended that we should use f-strings for string formatting whenever possible.
Let me know if you would like to change this, otherwise pr LGTM.

Reply:
Oh, thank you for the notice. I'll update it.